src/python/pants/pantsd/pailgun_server.py

```diff
@@ -270,13 +273,16 @@ def yield_and_release(time_waited):
         yield
     else:
       # We have to wait for another request to finish being handled.
-      NailgunProtocol.send_stderr(request, "Another pants invocation is running. Will wait {} for it to finish before giving up.\n".format(
+      self._send_stderr(request, "Another pants invocation is running. Will wait {} for it to finish before giving up.\n".format(
         "forever" if self._should_poll_forever(timeout) else "up to {} seconds".format(timeout)
       ))
+      self._send_stderr(request, "If you want to have multiple pants invocations at once, please "
+                                 "press Ctrl-C and run with PANTS_ENABLE_PANTSD=False in the "
```

Comment:
I would much much rather this be `--no-enable-pantsd`, since an env var would discourage putting `export PANTS_ENABLE_PANTSD=False` in `.bashrc` files:
```suggestion
                                 "press Ctrl-C and run with the --no-enable-pantsd flag"
```

Reply:
Sure, but if the user is running pants via a script which provides a repl, they wouldn't be able to add a command-line option. Either we say "use the command-line flag, or alternately the env var if you don't know where the command line is", or we can just say `PANTS_ENABLE_PANTSD=False`, which can be put at the start of any command line regardless of whether the user is invoking pants directly.

Reply:
I also don't quite understand why suggesting an env var discourages anything in `.bashrc` files?

Reply:
> `PANTS_ENABLE_PANTSD=False`, which can be put at the start of any command line regardless of whether the user is invoking pants directly.

That's a good point, ideally I would like users to use the CLI flag, so escalate to whoever owned the script that defined the CLI, but I agree that we don't want to charge users that _just want a repl_ with that responsibility.



> I also don't quite understand why suggesting an env var discourages anything in `.bashrc` files?

I meant "encourage", sorry. I would prefer the CLI flag to the env var because it's easy for users to say "okay, pantsd doens't work _now_, so I'm going to disable it in my .bashrc." and then forget to un-disable it forever. However, to your previous point, I now agree that the env var is probably the most convenient for our users, which is definitely what we should be doing.